app/models/user.js

```diff
@@ -58,6 +59,7 @@ module.exports = (sequelize, DataTypes) => {
     hooks: {
       afterValidate: (user, options) => {
         user.password = bcrypt.hashSync(user.password, bcrypt.genSaltSync(8), null);
+        user.lastInvalidation = moment();
```

Comment:
i'm not pretty sure if this will work fine if someone updates some other field, for example name.